msg.proto

```diff
@@ -99,4 +108,11 @@ message Msg {
   bytes write_file_sync_data = 131;
   uint32 write_file_sync_perm = 132;
   // write_file_sync_perm specified by https://godoc.org/os#FileMode
+
+  int32 net_port = 134;
```

Comment:
What is the reason this is 134 instead of 133?

Reply:
Fixed